src/hw/main.cpp

```diff
@@ -182,6 +184,25 @@ void createInventory(boost::asio::io_service& io,
             }
             break;
         }
+        else if (path.find("Raiser") != std::string::npos)
+        {
+            for (const auto& [property, value] : findIface->second)
+            {
+                try
+                {
+                    if (property.find("BOARD_INFO_AM") == 0)
+                    {
+                        pcieConfiguration.addBifurcationConfig(
+                            std::get<std::string>(value));
+                    }
+                }
+                catch (std::exception const& ex)
+                {
+                    log<level::ERR>("Error while reading FRU data",
```

Comment:
It looks like "Error while adding/storing bifurcation settings".

Reply:
Do you mean that this exception can be thrown only by `std::get<std::string>(value)`?

Reply:
I mean `addBifurcationConfig` doesn't actually "store". In my understanding message like "Error while adding/storing bifurcation settings" mean that the error appears when we try to apply changes (send to settings manager in this case).

Reply:
As I can see `addBifurcationConfig` doesn't throw any exceptions.
Probably this `try-catch` is excess in general. What do you think?

Reply:
It should not trow anything, but I prefer to keep just in case